### ISSUE TITLE
#Devil Immobility after reviving when succumbing in space

### ISSUE THREAD

**Issue by gorgonsaint** [tgstation/tgstation#24867]
[Note]: # (Please enter the commit hash and active testmerge numbers from the "Show Server Revision" verb if you can. If you believe the issue to be caused by a testmerge, please report it in its relative PR thread. State what the issue is from a "whats wrong" prospective. Issue reports should clearly allow maintainers to understand whats wrong and how to test/reproduce if that is not obvious. Avoid ambiguity. Start your issue report below both of these lines (or remove them))

[Admins]: # (If you are reporting a bug that occured AFTER you used varedit/admin buttons to alter an object out of normal operating conditions, please verify that you can re-create the bug without the varedit usage/admin buttons before reporting the issue.)

After reviving in space as a devil I couldn't move in any direction. My character was on a platform, but the game wouldn't respond to any of my movement keys.

**Comment by coiax** [comment]
@lordpidey

**Comment by lordpidey** [comment]
Were you in a humanoid or true devil form?

Also, this sounds like it MIGHT have to do with you being in zero gravity with nothing to push off of.

**Comment by gorgonsaint** [comment]
Humanoid, I had died before i could jaunt and when I revived I reverted to a human who couldn't use that ability. 
The next round I was given a pity devil antag and found that whenever i canceled my jaunt i couldnt movie unless I did a complete jaunt, and I looked like I was constantly on fire for the rest of the game. 
Much like when I revived in space. The only way I could move was to throw my pitchfork and fire balls and whenever I was on something stable I could no longer move.
